File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Double/System.Math/Double.Cos.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_211

	''' <summary>
	'''     Returns the cosine of the specified angle.
	''' </summary>
	''' <param name="d">An angle, measured in radians.</param>
	''' <returns>The cosine of . If  is equal to , , or , this method returns .</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function Cos(d As [Double]) As [Double]
		Return Math.Cos(d)
	End Function
End Module

File: src/Z.Core/System.Double/System.Math/Double.Cos.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Returns the cosine of the specified angle.
    /// </summary>
    /// <param name="d">An angle, measured in radians.</param>
    /// <returns>The cosine of . If  is equal to , , or , this method returns .</returns>
    public static Double Cos(this Double d)
    {
        return Math.Cos(d);
    }
}